flight/Modules/Stabilization/stabilization.c

```diff
@@ -219,6 +220,15 @@ static void stabilizationTask(void* parameters)
 		dT = PIOS_DELAY_DiffuS(timeval) * 1.0e-6f;
 		timeval = PIOS_DELAY_GetRaw();
 		
+		// exponential moving averaging (EMA) of dT to reduce jitter; ~200points
+		// to have more or less equivalent noise reduction to a normal N point moving averaging:  alpha = 2 / (N + 1)
+		dT_filtered = 0.01f * dT + (1.0f - 0.01f) * dT_filtered;
```

Comment:
This is a good idea. I think it would make sense to also pass this to `pid_apply_setpoint` instead of `dT`.